src/main/java/homework1/Main.java

```diff
@@ -0,0 +1,13 @@
+package homework1;
+
+public class Main {
+
+    public static String wordAndNumber (String word){
+
+        if (word.equals("ошибка")) {
+           return "\u001b[31m" + " ТРЕВОГА";
+        } else return word + ": " + word.length();
```

Comment:
Doesn't comply with AC, as "letters" word in the end needed.

Reply:
I'm sorry to say  I do not really understand what AC is. I added the word  letters to the end of this String, I hope I did that right.